perf.go

```diff
@@ -166,6 +174,9 @@ func (m *PerfMap) Stop(cleanup MapCleanupType) error {
 			err = fmt.Errorf("%s: %w", err.Error(), errTmp)
 		}
 	}
+
+	m.manager.wg.Done()
```

Comment:
I think this is redundant now, as the manager is the only one calling this function synchronously